tests/keras/layers/convolutional_test.py

```diff
@@ -279,7 +279,6 @@ def test_separable_conv_1d():
                                                           batch_input_shape=(None, 5, None))])
 
 
-@pytest.mark.skipif(K.backend() == 'theano', reason='Theano does not support it yet')
 @keras_test
 def test_separable_conv_2d():
```

Comment:
Please enable the `test_separable_conv_1d` test as well (if possible).

Reply:
@fchollet, `separable_conv1d` is not yet implemented. I will add the function. Also, the layer tests are enabled now.